src/ntptimeclient.cpp

```diff
@@ -150,3 +150,49 @@ bool NTPTimeClient::UpdateClockFromWeb(WiFiUDP * pUDP)
 
     return true;
 }
+
+// This is underdesigned for general use. It's very specifically a
+// throwaway meant only for debugging. It's not really NTP related.
+//
+// It contains some ESP32-specific goo, but that's allowed elsewhere.

```

Comment:
I don't think you need to label this as a throwaway. We've recently concluded that this is a developer-oriented project "with one exception", so a developer-oriented feature is a feature.

I'm also not sure it's necessary to include "defensive ponderings" about implementation decisions in the source code comments. Where appropriate, comments about implementation decisions can be added to the PR when it's opened.
(I'm aware there may be other places in the code where these comments exist, but up to this point they've been grandfathered as artifacts from before the project was made publicly available.)